talent/onboarding/Non-EU_residents.md

```diff
@@ -88,4 +88,41 @@ If the candidate has a family, he needs to [ask for an appointment](https://w6.s
 
 With this document the candidate has to go to the closest clinic (Centro de Salud) to their house and ask for the Health card (Tarjeta sanitaria).
 
+## Spanish Residence Permit Renewal 
+
+We have to apply to renew the resident card within 60 days before the card expires. See below a list of documents we will need to renew the permit. 
```

Comment:
@estherrgarcia is this something each employee or the company will keep track of?

Reply:
The company will keep track of that because the visa is linking to the Company.